packages/block-editor/src/components/block-navigation/style.scss

```diff
@@ -121,8 +121,8 @@
 
 	.block-editor-block-icon {
 		align-self: flex-start;
-		margin-right: ( $grid-unit-05 * 2.5 ); // 10px is off the 4px grid, but required for visual alignment between block name and subsequent nested icon
```

Comment:
CC: @jameskoster, the icon was scaled down to 20px, so it didn't show true pixels. I shifted the metrics around a bit, but would appreciate you taking a look when you have a chance, thank you.

Reply:
<img width="196" alt="Screenshot 2021-05-28 at 10 41 48" src="https://user-images.githubusercontent.com/846565/119964662-56183900-bfa1-11eb-9e2e-24edc145dba2.png">

Looks good to me.